airbyte-ci/connectors/pipelines/pipelines/builds/python_connectors.py

```diff
@@ -76,29 +89,58 @@ async def _provision_builder_container(self, base_container: Container) -> Conta
 
         return builder.with_exec(["pip", "install", "--prefix=/install", "."])
 
+    def _get_build_customization_module(self) -> Optional[ModuleType]:
```

Comment:
❓ We will be doing this for java connectors correct? If so lets move this to a more generic location